apps/aefate/src/aefa_fate.erl

```diff
@@ -295,6 +306,16 @@ step([I|Is], EngineState0) ->
             catch_protected(Err, EngineState0)
     end.
 
+break_or_step(Is, ES) ->
+    case aefa_engine_state:debugger_status(ES) of
+        break ->
+            InstructionsCount = length(aefa_engine_state:current_bb_instructions(ES)),
+            InstructionsSkip = InstructionsCount - length(Is),
+            {break, aefa_engine_state:set_skip_instructions(InstructionsSkip, ES)};
```

Comment:
Consider adding instruction counter in dbg state

Reply:
Something like this 4e0f209ed631428d104bdafee71d65cf42aa62bf ?

Reply:
yes